core/thirdpartyapp/subgraph.py

```diff
@@ -0,0 +1,39 @@
+import logging
+from core.request_helper import RequestException, RequestHelper
+from core.thirdpartyapp import config
+
+
+def fetch_nft_pass_wallets():
+    results = []
+    while True:
+
+        requests = RequestHelper()
+        session = requests.get_session()
+        data = None
+        try:
+            data = session.post(
+                config.NFT_PASS_SUBGRAPH_URL,
+                json={
+                    "query": "{nfts(first: 1000, skip: "
+                    + str(len(results))
+                    + ") {id owner tokenId}}"
+                },
+            ).json()
+        except:
+            logging.error("Error fetching nft pass wallets")
+            break
+
+        if len(data["data"]["nfts"]) == 0:
+            break
+        for item in data["data"]["nfts"]:
+            results.append(item)
```

Comment:
why you append item and then process owner and did n't get owner right here?

Reply:
because the count of NFTs and the count of holders are different, I needed the whole data length to ensure all data returned.
I'll optimize it a littile bit in the next commit.